### DISCUSSION TITLE
Load the whole project

### DISCUSSION THREAD

**Discussion by CyberDruga** [nvim-java/nvim-java#193] [Q&A]
is there a way to make jdtls load the whole project instead of just the project the opened file resides in ? I'm having a particular problem where if I just open the file I want I get a bunch of "the import x cannot be resolved", but when I first open a file from that import, and then the file I want everything works.

**Comment by s1n7ax** [comment]
I have no clue tbh. but you can make the root folder a git repository so jdtls will consider the closest git repo as the root dir
